roles/bootstrap-os/tasks/bootstrap-fedora.yml

```diff
@@ -44,3 +44,14 @@
   become: true
   when:
     - not is_atomic
+  with_items:
+    - libselinux-python
+    - python-pip
+
+- name: Install required python modules
```

Comment:
Please don't install things for fedora if you are working on Oracle.

Reply:
Not related to the Oracle work. #1032

Reply:
That issue complains about a package name being wrong for Fedora, it doesn't explain if this package is even necessary for anything.

Reply:
Back in November when I worked on this PR, I ran into the exact issue he described; and using https://github.com/kubernetes-sigs/kubespray/issues/1032#issuecomment-280394442 resolved the issue.